vllm/entrypoints/utils.py

```diff
@@ -231,3 +234,17 @@ def show_filtered_argument_or_group_from_help(parser, subcommand_name):
             print(f"\nNo group or parameter matching '{search_keyword}'")
             print("Tip: use `--help=listgroup` to view all groups.")
             sys.exit(1)
+
+
+def get_max_tokens(max_model_len: int, request: Union[ChatCompletionRequest,
+                                                      CompletionRequest],
+                   input_length: int, default_sampling_params: dict) -> int:
+
+    max_tokens = getattr(request, "max_completion_tokens", request.max_tokens)
```

Comment:
This will fix the test error:
```suggestion
max_tokens = getattr(request, "max_completion_tokens", None) or request.max_tokens
```
If the attribute exists but is None, getattr will return None and not the default.